services/src/server.rs

```diff
@@ -61,6 +61,7 @@ pub async fn start_server(shutdown_rx: Option<Receiver<()>>) -> Result<()> {
         .or(handlers::wms::wms_handler(workflow_registry.clone()))
         .recover(handle_rejection);
 
+    #[allow(clippy::option_if_let_else)]
```

Comment:
can't use option.map_or_else here because I need the handler variable in both cases and with map_or_else it would be captured by the first closure

Reply:
Yeah, maybe this lint is too strict, we'll have to investigate it.